app/styles/mixins/_layout.scss

```diff
@@ -0,0 +1,6 @@
+@mixin l-bottom {
```

Comment:
not sure. I couldn't find any BEM official guidance for prefixing stuff like this.  I think it's up to us however we want to define our prefixes.